### ISSUE TITLE
Fedora 27 massive rebuild

### ISSUE THREAD

**Issue by sztsian** [FZUG/repo#266]
https://fedorapeople.org/groups/schedule/f-27/f-27-all-tasks.html

Fedora 27 Beta is planned on 2017-09-19, and GA around 2017-10-24. We should rebuild all packages against Fedora 27 by that time.

**Comment by sztsian** [comment]
Currently blocked by this bug https://bugzilla.redhat.com/show_bug.cgi?id=1498207

**Comment by sztsian** [comment]
Massive rebuilt against Fedora-27-20171005.n.0 (with RPMFusion repo disabled) is done and result now pending publish on the repo server.
Corresponding commit which the massive rebuild run against: 71de37db4a5bc92231a748cc4f5358d2cdc9350b

**Comment by hooklab** [comment]
well done

**Comment by sztsian** [comment]
已经完成且发布了。部分包编译失败。Deepin 相关的包大多数已经进入 Fedora 官方源所以本次没编译。
对于具体包的问题**请另开 issue 报告**。

Done and published.
Some package failed to build. Deepin related are mostly in Fedora official repo.
I am closing this ticket as fixed. Feel free to open separate issue for specific package.
https://repo.fdzh.org/FZUG/free/27/
